Gifski/EditVideoViewController.swift

```diff
@@ -248,12 +251,23 @@ final class EditVideoViewController: NSViewController {
 			self.estimateFileSize()
 		}
 
+		loopButton.onAction = { [weak self] _ in
```

Comment:
Instead of manually handling this, in Interface Builder, you can bind the checkbox directly to the userdefaults key.

Reply:
Sure thing. I was trying to keep the same style as other properties' handling so changes will be focused on changed functionality.

Reply:
@sindresorhus wow didn't know that! great idea